NewForce-Cohort-9/newforce-c9-tabloidmvc-TabloidMVC
# View Tags ticket 13

Description: 
Completed trello ticket #13 view all tags. This PR adds view all tags functionality for admins. 

Type of Change: New Feature

Testing Instructions:

1. Clone repo down into a local repository and make sure you have the database added
2. git fetch --a in the main branch to retrieve my branch named ks-branch1
3. git checkout ks-branch1 to switch to my branch and open up Visual Studio. Run the application in a browser window.
Use admin@example.com
You should see the 'Home' page with a navigation bar at the top of the page.
Click the option 'Tag Management'. You will be directed to a new page that lists all of the tags in alphabetical order.
It should look as such: C#, Cyclopean Terrors, Family, JavaScript.
Checklist:

I have opened ks-branch1 in my local repository
I have ran my application in a browser window to see the 'Tag Management' on the navbar
After logging in, and clicking on 'Tag
 Management' in the navbar, it directs me to the page where you can view all of the available tags in alphabetical order
I have added test instructions that can be followed to test this code.
